bootloader/src/pyi_win32_utils.c

```diff
@@ -444,20 +493,29 @@ pyi_win32_utf8_to_mbs(char * dst, const char * src, size_t max)
 int
 pyi_win32_mkdir(const wchar_t *path)
 {
-    wchar_t stringSecurityDesc[] = // ACE String :
-        L"D:" // DACL (D) :
-        L"(A;" // Authorize (A)
-        L";FA;" // FILE_ALL_ACCESS (FA)
-        L";;S-1-3-4)"; // For the current directory owner (SID: S-1-3-4)
```

Comment:
Why you are no longer using wide strings here? AFAI see, this adds another copy-operation in pyi_win32_get_user_sid

Reply:
Yeah, I abused the conversion operation to create a copy of the string that can be freed using `free()` instead of `LocalFree()`. But since `_pyi_win32_get_user_sid` is  a private function used only here, I've revised this so that original wide SID is returned and freed using `LocalFree()`. 

So in the revised version, `stringSecurityDesc` is again wide string.